tests/unit/test_helpers.py

```diff
@@ -215,3 +221,11 @@ def test_group_by_signature(curated_signature, non_curated_signature):
     result = group_by_signature([curated_signature, non_curated_signature])
     expected_result = "94fc2b0a-dc17-42c2-bae3-ca0024079e52"
     assert result == expected_result
+
+
+def test_train_validation_data_split(curated_signature):
+    split_result = train_validation_split([curated_signature], 0.9)
+
+    assert len(split_result) == 2
+    assert len(split_result[0]) == 1
+    assert len(split_result[1]) == 0
```

Comment:
I think you can write a better test, given you're fixing the seed, so everything should be deterministic.

Reply:
there are plenty of ways this can go wrong that you're not testing. For example, if you had written a more detailed test here, you would have realized that what it returns is not what you think it returns (mismatch between return types and assumed return types in api).